wand/image.py

```diff
@@ -1158,11 +1158,12 @@ class BaseImage(Resource):
 
     __slots__ = '_wand',
 
-    def __init__(self, wand):
+    def __init__(self, wand, options={}):
```

Comment:
We can remove this, as `self.options.update` should occur in `Image._preamble_read()`